docs/preview/02-Features/writing-different-telemetry-types.md

```diff
@@ -429,6 +429,38 @@ logger.LogMetric("Invoice Received", 133.37, telemetryContext);
 
 ## Requests
 
+### Incoming Azure Service Bus requests
+Requests allow you to keep track of incoming Azure Service Bus messages on a queue or topic and how the message was processed.
```

Comment:
```suggestion
Requests allow you to keep track of incoming Azure Service Bus messages on a queue or topic.
```
I think we can remove the 'and how the message was processed' part, as that is no longer the responsability of the request that we're logging.